Source/dom/import.js

```diff
@@ -0,0 +1,35 @@
+import { wrapNativeObject } from './wrapNativeObject'
+
+function _importWithImporter(data, importer) {
+  importer.prepareToImportFromData(data)
+  return wrapNativeObject(importer.importAsLayer())
+}
+
+function _importSVG(data) {
+  return _importWithImporter(data, MSSVGImporter.svgImporter())
+}
+function _importPDF(data) {
+  return _importWithImporter(data, MSPDFImporter.pdfImporter())
+}
+function _importEPS(data) {
+  return _importWithImporter(data, MSPDFImporter.epsImporter())
+}
+
+export const layerFromData = (data, type) => {
+  switch (type) {
+    case 'svg':
+      return _importSVG(data)
+    case 'pdf':
+      return _importPDF(data)
+    case 'eps':
+      return _importEPS(data)
+    default:
```

Comment:
it'd be super easy to handle bitmaps as well: `return Image({image: data})`

Reply:
Shouldn't Image accept a Buffer for its data? That doesn't appear to be the case right now.

Reply:
Ah I thought I did it but I might not have pushed it

Reply:
If not, I've added it in to try it out.

Reply:
found my branch: https://github.com/BohemianCoding/SketchAPI/pull/372